services/compliance/continuous_compliance_policy/continuous_compliance_policy.go

```diff
@@ -47,8 +47,8 @@ func (service *Service) GetAll() (*[]ContinuousCompliancePolicyResponse, *http.R
 	return v, resp, nil
 }
 
-func (service *Service) Create(body *ContinuousCompliancePolicyRequest) (*ContinuousCompliancePolicyResponse, *http.Response, error) {
-	v := new(ContinuousCompliancePolicyResponse)
+func (service *Service) Create(body *[]ContinuousCompliancePolicyRequest) (*[]ContinuousCompliancePolicyResponse, *http.Response, error) {
+	v := new([]ContinuousCompliancePolicyResponse)
```

Comment:
is that how api expect? to pass the request as a list? interesting... I wonder if it was made for a creation of multiple policies all at once...
just double check this because it will require more refactoring on the provider side (schema change)

Reply:
Yes, this why i got errors from API "bed request".